mesatee_services/fns/sgx_trusted_lib/src/global.rs

```diff
@@ -55,7 +57,7 @@ pub fn save_file_for_file_owner(
 }
 
 pub fn register_trusted_worker_statically() {
-    for _i in 0..10 {
+    for _i in 0..23 {
```

Comment:
The number is the max concurrent running tasks at the same time for each worker.  It doesn't need to be changed when you add new workers.